### ISSUE TITLE
the changes do not show after reloading textures or changes on the options

### ISSUE THREAD

**Issue by OlaPessoa** [GirafiStudios/BetterTitleScreen#33]
i'm not saying i'm trying to reload the resourcepacks to reload the config, THE MOD DOESNT WORK AFTER RELOADING RESOURCEPACKS, the title screen gets back to normal, i'm using version 1.13.2 for 1.20.1

**Comment by OlaPessoa** [comment]
PLEASE NEXT TIME WAIT FOR A RESPONSE, IF YOU OPEN A ISSUES TAB USING GITHUB THAT IMPLIES YOU'RE WILLING TO WAIT FOR PEOPLE TO RESPOND WITHIN A AMOUNT OF TIME AND ONE DAY AINT THE PROPPER AMOUNT OF TIME MAN.

**Comment by GirafiStudios** [comment]
If you restart the game after having reloaded the resourcepack, does it work correctly again then?

**Comment by OlaPessoa** [comment]
> If you restart the game after having reloaded the resourcepack, does it work correctly again then?

yes it does, but that's the entire point of the bug report, why would i want to use your mod instead of just fancy menu, if your mod does work but as soon as you tweak a setting or reload the resource packs the screen goes back to default??
